gprofiler/profilers/java.py

```diff
@@ -730,8 +829,13 @@ def __init__(
         self._pid_to_java_version: Dict[int, Optional[str]] = {}
         self._kernel_messages_provider = get_kernel_messages_provider()
         self._enabled_proc_events_java = False
+        self._java_jvm_flags_to_retrieve = self._init_java_flag_retrieval(java_jvm_flags_to_retrieve)
```

Comment:
Per https://github.com/Granulate/gprofiler/pull/660/files#r1096566312
```suggestion
        self._collected_jvm_flags = self._init_java_flags_collection(java_collect_jvm_flags_to)
```

Also rename to `--java-collect-jvm-flags=all`, `--java-collect-jvm-flags=default`.

Reply:
> Also rename to --java-collect-jvm-flags=all, --java-collect-jvm-flags=default.

@uriariel you didn't do this one and now I see [I commented it again](https://github.com/Granulate/gprofiler/pull/660#discussion_r1103750687). Please make sure you go over ALL review comments before re-requesting review.